server/views/pages/recommendations/confirmationPartA.njk

```diff
@@ -19,16 +19,55 @@
                 titleText: pageHeadings[page.id],
                 html: html
             }) }}
+
             <h2 class='govuk-heading-m govuk-!-margin-top-6'>What happens next</h2>
-            <ol class='govuk-list govuk-list--number'>
+            <ol class='govuk-list govuk-list'>
                 <li>
-                    <a class='govuk-link' href='/recommendations/{{ recommendation.id }}/documents/part-a?crn={{ recommendation.crn }}' download data-analytics-event='click_cta' data-analytics-event-category='download_part_a_click'>
-                        Download the Part A.</a>
+                <div>
+                    <a href='/recommendations/{{ recommendation.id }}/documents/part-a?crn={{ recommendation.crn }}' download data-analytics-event='click_cta' data-analytics-event-category='download_part_a_click' class='govuk-button govuk-button--secondary'>Download the Part A</a>
+                </div>
                 </li>
-                <li>Double-check that everything is correct.</li>
-                <li>Fill in any missing details from question 25 onwards.</li>
-                <li>Upload the Part A to NDelius.</li>
-                <li>Update OASys with any new risk information.</li>
+                <li id=part><h2 id="part-a-confirmaton-number-bullet">➊</h2>  Send this Part A to PPCS with the supporting documents. This table shows what documents you need.<br><br>
+               <table class="govuk-table">
```

Comment:
This looks like it could do with some formatting, removal of blank lines etc.

Reply:
formatting done 👍